### DISCUSSION TITLE
disable specific rules for cdk project

### DISCUSSION THREAD

**Discussion by ghost** [Checkmarx/kics#6004] [Q&A] [answered]
Hi team,

Here says rules can be disabled by putting comments in the code https://docs.kics.io/latest/running-kics/#using_commands_on_scanned_files_as_comments.

But with cdk project the code will be translated to either json or yaml and kics scans the translated json or yaml files. the comments solution will not work for cdk. Is there a way to specify and disable specific  rules in kics configuration?

**Comment by cx-miguel-dasilva** [comment, marked as answer]
Hi @MayaShenFraedom, hope you are doing great!

At the moment KICS does not support excluding security queries for selected technologies. Maybe you can use the flags --exclude-queries or --exclude-results . You can find more information regarding the flags [here](https://docs.kics.io/latest/commands/).

Hope I was able to help.
Let us know if you need anything else 🙂
